### ISSUE TITLE
Shorter if syntax 

### ISSUE THREAD

**Issue by qiufeihai** [dylanaraps/pure-bash-bible#112]
[[ 1 == 1 ]] && { command exit 1 } || { this command will exec  }
how to avoid???

**Comment by linux478** [comment]
I do not understand what you are trying to tell.

**Comment by Skizo4** [comment]
Thank you very much.

**Comment by TedHartDavis** [comment]
I am *so confused* by this.

**Comment by TinCanTech** [comment]
`Missing terminating quote, bracket or keyword`

**Comment by ghost** [comment]
I'm sure you found a solution by now, but just in-case you didn't:

You have two syntax errors, because you're missing the `;` before the final `}` in each command group. The typical way to group commands (without a subshell) is `{ like this; }`. Grouping a single command is redundant.

The `COMMAND && COMMAND || COMMAND` approach does not equate to `if`, `then`, `else`, although it _can_ work in a similar way. Likewise, `COMMAND || COMMAND && COMMAND` does not equate to the opposite.

Why? Because what the first example _actually_ means is that the 2nd command will execute if the 1st one succeeds, or the 3rd command will execute if the 1st command fails. However, if the 1st command succeeds _and_ the 2nd command also happens to fail, then all _three_ commands wind up being executed regardless. Understanding this is very important, as you can wind up doing some pretty stupid stuff otherwise.

Furthermore, the exit statuses are _not_ contained, unlike with an `if` statement, which is pretty important if it's the last command in a script or you're checking `$?`.

**Comment by YasserKa** [comment]
"Shorter if syntax" should be removed to stop spreading miss-information.

More context can be found here: http://mywiki.wooledge.org/BashPitfalls#cmd1_.26.26_cmd2_.7C.7C_cmd3

**Comment by ghost** [comment]
Agreed, it's misleading and incorrect. I see what the author meant, but it's important to be accurate when it comes to complicated matters like programming, IMO. Logical operators _can_ be used _similar to_ an `if` statement, in a simple way, but it's _not_ technically or actually a replacement or "shorter" version.
